langmeta/sql/src/main/scala/org/langmeta/sql/Index.scala

```diff
@@ -60,6 +60,24 @@ object Index {
           val symbolStmt = tableInsertStmt("symbol")
           val syntheticStmt = tableInsertStmt("synthetic")
 
+          def persistBatch = {
```

Comment:
Let's make this `def persistBatch()` and call this with `persistBatch()`, since the method is side-effecting. Otherwise LGTM.

Reply:
Great catch, thanks.